app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/cases_list.scala.html

```diff
@@ -41,17 +45,20 @@
                 <th scope="col">@messages("cases.table.elapsed-days")</th>
             </tr>
         </thead>
-        <tbody id="@{id}-body">
-            @for((c: Case, index: Int) <- cases.results.zipWithIndex) {
-                @defining(c.application) { a: Application =>
+        <tbody id="@{
+            id
+        }-body">
+        @for((c: Case, index: Int) <- cases.results.zipWithIndex) {
+            @defining(c.application) { a: Application =>
+                @if(a.isBTI) {
```

Comment:
You might be able to do this differently.  There is obviously a lot of common code in the two branches.   If you look at the `Application` trait there is a method `businessName` which return the name for either application type.   You could create a similar `goodName` method.

Then (I think) the only difference to factor out would be the rendering of the type name and status which you could create a component for.

Reply:
Have tidied this up a lot, and it's much simpler now.